bigtable/docs/snippets.py

```diff
@@ -160,6 +170,183 @@ def test_bigtable_create_app_profile():
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    # Define the GC policy to retain only the most recent 2 versions
+    max_versions_rule = column_family.MaxVersionsGCRule(2)
+
+    column_family_obj = table.column_family('cf1', max_versions_rule)
+    column_family_obj.create()
+
+    # [END bigtable_create_MaxVersionsGCRule]
+    rule = str(column_family_obj.to_pb())
+    assert "max_num_versions: 2" in rule
+    column_family_obj.delete()
+
+
+def test_bigtable_create_MaxAgeGCRule():
+    # [START bigtable_create_MaxAgeGCRule]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    # Define the GC rule to retain data with max age of 5 days
+    max_age_rule = column_family.MaxAgeGCRule(datetime.timedelta(days=5))
+
+    column_family_obj = table.column_family('cf1', max_age_rule)
+    column_family_obj.create()
+
+    # [END bigtable_create_MaxAgeGCRule]
+    rule = str(column_family_obj.to_pb())
+    assert "max_age" in rule
+    assert "seconds: 432000" in rule
+    column_family_obj.delete()
+
+
+def test_bigtable_create_GCRuleUnion():
+    # [START bigtable_create_GCRuleIntersection]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    max_versions_rule = column_family.MaxVersionsGCRule(2)
+    max_age_rule = column_family.MaxAgeGCRule(datetime.timedelta(days=5))
+
+    union_rule = column_family.GCRuleUnion([max_versions_rule, max_age_rule])
+
+    column_family_obj = table.column_family('cf1', union_rule)
+    column_family_obj.create()
+
+    # [END bigtable_create_GCRuleIntersection]
+    rule = str(column_family_obj.to_pb())
+    assert "union" in rule
+    assert "max_age" in rule
+    assert "seconds: 432000" in rule
+    assert "max_num_versions: 2" in rule
+    column_family_obj.delete()
+
+
+def test_bigtable_create_GCRuleIntersection():
+    # [START bigtable_create_GCRuleUnion]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    max_versions_rule = column_family.MaxVersionsGCRule(2)
+    max_age_rule = column_family.MaxAgeGCRule(datetime.timedelta(days=5))
+
+    intersection_rule = column_family.GCRuleIntersection([
+        max_versions_rule,
+        max_age_rule
+        ])
+
+    column_family_obj = table.column_family('cf1', intersection_rule)
+    column_family_obj.create()
+
+    # [END bigtable_create_GCRuleUnion]
```

Comment:
should be `...GCRuleIntersection` tag
Follow the tag format above. (if different format is chosen)